MAINTAINERS

```diff
@@ -79,7 +79,8 @@
 			"ripcurld",
 			"sam-thibault",
 			"samwhited",
-			"thajeztah"
+			"thajeztah",
+			"thompson-shaun"
```

Comment:
I don't recall; does TOML allow trailing commas in the last entry? If so , I could add one here to reduce code-churn.

Reply:
Looks like it does; so let me update once more; https://github.com/toml-lang/toml/issues/246#issuecomment-57919189

Reply:
Looks like it does!

> Arrays can span multiple lines. A terminating comma (also called a trailing comma) is permitted after the last value of the array. 

https://toml.io/en/v1.0.0#array

Reply:
Yup! I updated to add one, might as well have that in place already 😄 (I didn't update the other list further up yet though as there was no need yet)